src/main/java/org/pulseus/auth/service/rest/controller/CSVController.java

```diff
@@ -42,8 +38,28 @@ public void addusers() throws Exception {
 		BufferedReader br = null;
 		String line = "";
 		String cvsSplitBy = ",";
+		
 
 		br = new BufferedReader(new FileReader("/opt/pulse/users.csv"));
```

Comment:
This would be an excellent candidate for a properties value. Hard-coding filenames isn't awesome